code/game/gamemodes/clock_cult/clock_structure.dm

```diff
@@ -160,7 +160,7 @@
 	if(is_servant_of_ratvar(user) || isobserver(user))
 		var/powered = total_accessable_power()
 		var/sigil_number = LAZYLEN(check_apc_and_sigils())
-		to_chat(user, "<span class='[powered ? "brass":"alloy"]'>It has access to <b>[powered == INFINITY ? "INFINITY":"[powered]"]W</b> of power, \
+		to_chat(user, "<span class='[powered ? "brass":"alloy"]'>It has access to <b>[powered == INFINITY ? "INFINITE UNITS":"[DisplayPower(powered)]"]</b> of power, \
```

Comment:
I can't think of a better way to word this than `INFINITE UNITS` but I don't like it.

Reply:
`UNLIMITED AMOUNTS`?

Reply:
"An infinite amount of power", maybe?

Reply:
How about endless amount?

Reply:
[`Unlimited power!`](https://youtu.be/Q_7KaMDHoGs?t=183)